app/build.gradle

```diff
@@ -49,4 +62,8 @@ dependencies {
 
     // Image Request
     implementation 'com.github.bumptech.glide:glide:4.11.0'
+    annotationProcessor 'androidx.room:room-compiler:2.2.5'
+
+    // Rx Java/Android
+    implementation 'io.reactivex.rxjava3:rxandroid:3.0.0'
 }
```

Comment:
Thêm dòng trống vào cuối file. Ô có thể cài android studio thêm blank line cuối file như sau `File | Settings | Editor --> Ensure blank line before end of file on Save`